### ISSUE TITLE
Flexible parametric competing risks models

### ISSUE THREAD

**Issue by mattwarkentin** [chjackson/flexsurv-dev#93]
Hi @chjackson,

Do you have any thoughts about whether this package would ever add support for being able to use flexible parametric models to directly model the cause-specific cumulative incidences? In particular, I'm wondering if the methods described in [this](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6175038/) paper (available in Stata via `stmp2cr`) would be a good candidate feature for `flexsurv`.

**Comment by chjackson** [comment]
I agree that regression on cumulative incidences would be a good fit for the package, as this is a common approach to competing risks models.  I'd implement it if I needed this class of models in my own work, which I don't at the moment but it's not out of the question that I will in the future!  

I've also not reviewed what other software is available for these models -  someone else may have done something related in R.
